forest/src/cli/chain_cmd.rs

```diff
@@ -21,7 +21,8 @@ pub enum ChainCommands {
     /// Retrieves and prints out the block specified by the given CID
     #[structopt(about = "<Cid> Retrieve a block and print its details")]
     Block {
-        #[structopt(short, help = "Input a valid CID")]
+        #[structopt(short)]
+        /// Input a valid CID
```

Comment:
Let's use the same pattern everywhere, i.e. doc comment first, then structopt and field definition.
```suggestion
        /// Input a valid CID
        #[structopt(short)]
```